iOS_homework_1.playground/Pages/Fightning.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,278 @@
+import Foundation
+
+protocol Student {
+    var name: String { get set }
```

Comment:
```suggestion
    var name: String { get }
```
не изменяемое свойство